week_2/pet.rb

```diff
@@ -90,7 +95,11 @@ def food_cost(days)
   # snake = Pet.new(name: 'python', animal_type_id: 4, food_consumed_per_day: 0.3)
   # Pet.cost_to_feed([cat, dog, fish, snake], 6) will return 6180.0
   def self.cost_to_feed(pets, days)
-    raise NotImplementedError # TODO
+    ttpd=0
+    for pet in pets
+        ttpd+=pet.food_cost(days)
+     end
+    return ttpd
```

Comment:
[reduce](https://medium.com/@james.a.hughes/using-the-reduce-method-in-ruby-907f3c18ae1f) method cab be used here.

Reply:
changes are made, please verify it